crates/rpc/api/src/lib.rs

```diff
@@ -133,6 +162,14 @@ pub trait StrataAdminApi {
 #[cfg_attr(not(feature = "client"), rpc(server))]
 #[cfg_attr(feature = "client", rpc(server, client))]
 pub trait StrataSequencerApi {
+    /// Get the last broadcast entry
+    #[method(name = "strata_getLastBroadcastEntry")]
+    async fn get_last_broadcast_entry(&self) -> RpcResult<Option<L1TxEntry>>;
+
+    /// Get the broadcast entry by its id
+    #[method(name = "strata_getBroadcastEntryByIdx")]
+    async fn get_broadcast_entry_by_idx(&self, idx: HexBytes32) -> RpcResult<Option<L1TxEntry>>;
```

Comment:
Maybe should be on the rollup client namespace?  Since the sequencer won't be directly connected to the bitcoin client in the future.

Reply:
We can move this later when sequencer don't have direct connection.